src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -32,5 +32,4 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
     public Optional<List<? extends ReadOnlyPerson>> getRelevantPersons() {
         return Optional.ofNullable(relevantPersons);
     }
-
```

Comment:
Be careful of unnecessary or accidental changes here as it can cause unnecessary merge conflict.